openlabcmd/openlabcmd/cli.py

```diff
@@ -124,6 +124,11 @@ def _add_ha_service_cmd(self, parser):
         cmd_ha_service_list = cmd_ha_service_subparsers.add_parser(
             'list', help='List all services.')
         cmd_ha_service_list.set_defaults(func=self.ha_service_list)
+        cmd_ha_service_list.add_argument(
+            '--node', help='Filter the services with the specified node.')
+        cmd_ha_service_list.add_argument(
+            '--role', choices=['master', 'slave', 'zookeeper'],
+            help='Filter the services with the specified node role.')
         # openlab ha service get
         cmd_ha_service_get = cmd_ha_service_subparsers.add_parser(
             'get', help='Get a services.')
```

Comment:
Not relate with this pr: Get a service?

Reply:
Please take a look at the pr commit message. I did a lot change in this pr.

Reply:
I mean this should be 'Get a service'.